### ISSUE TITLE
Update omits empty object

### ISSUE THREAD

**Issue by suevalov** [substantial/updeep#29]
I have the following situation:

```js
const store = {
   widgets: {
     1: { ... },
     2: { ... }
   }
}
```
Each widget has the following structure: 

```js
const widget = {
   id: number,
   title: string,
   displayParameters: Object
   ...
}
```

Sometimes `displayParameters` object may be empty.

So, the problem is that if `displayParameters` is an empty object, it disappears after `updeep`.

```js
const newWidget = {
   id: 123,
   title: 'new one',
   displayParameters: {}
};
store = updeep({
  widgets: {
    [newWidget.id]: newWidget
  }
}, store)

/*
store: {
 widgets: {
   ...
   123: {
     id: 123,
     title: 'new one' 
   }
  }
}
*/
``` 

I have checked this issue and wrote the test - the same result. Not sure if it's correct behaviour.

```js

it('it should not omit empty object', () => {
    const store = {
      widgets: {
        1: {
          title: '123',
          displayParams: {
            index: 1
          }
        }
      }
    };

    const result = u({
      widgets: {
        123: {
          title: 'new widget',
          displayParams: {}
        }
      }
    }, store);

    expect(result).toEqual({
      widgets: {
        1: {
          title: '123',
          displayParams: {
            index: 1
          }
        },
        123: {
          title: 'new widget',
          displayParams: {}  // test is failed here
        }
      }
    });
  });

```

**Comment by aaronjensen** [comment]
Thanks for the report and the test. This is fixed in 0.7.1. The behavior was actually intentional, but it's overly pedantic and surprising, I think.

**Comment by suevalov** [comment]
Wow, it was quick. Thanks!
